docs/source/tutorial/tutorial-introduction.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: "0. Introduction"
+---
+
+Welcome! This tutorial demonstrates adding the Apollo iOS SDK to an app to communicate with a GraphQL server. In this tutorial you will learn how to:
+
+* Download a schema
+* Run code generation
+* Write queries and mutations
+* Handle errors
+* Handle Pagination
+* Authenticate with a backend server
+* Use subscriptions to receive real-time updates
+
+The tutorial uses an instance of [Apollo Server](https://www.apollographql.com/docs/apollo-server/) hosted on Heroku, and an [Apollo Studio Sandbox](https://studio.apollographql.com/sandbox/explorer?endpoint=https%3A%2F%2Fapollo-fullstack-tutorial.herokuapp.com%2Fgraphql&_gl=1*d2b33m*_ga*MzQ1Mzc2MDg1LjE2ODE3NjA2ODc.*_ga_0BGG5V2W2K*MTY4Mjk2MjIwMi4yNS4xLjE2ODI5NjIyMzIuMC4wLjA.) which connects to that server. If you're curious about how to build your own server, see the [Apollo full-stack tutorial](https://www.apollographql.com/docs/tutorial/introduction/).
+
+> If you encounter any issues during the tutorial, feel free to ask questions by either [opening an issue on our GitHub repo](https://github.com/apollographql/apollo-ios/issues) or [stopping by our community forums for help](https://community.apollographql.com).
```

Comment:
Hmmm that's a good question, originally the tutorial was pointing to the apollo-ios repo, but I think it may be better to direct issues to the iOSTutorial repo since any issue that comes up is likely directly related to the tutorial.

Reply:
I think so too.